src/app/page.tsx

```diff
@@ -1,11 +1,12 @@
-"use client";
-
 import OptionsList from "@/components/OptionsList";
+import { getAvailableOptions } from "@/lib/api";
+
+export default async function Home() {
+  const options = await getAvailableOptions();
 
-export default function Home() {
   return (
     <main className="flex min-h-screen flex-col items-center justify-between p-24">
-      <OptionsList />
+      <OptionsList data={options?.data} />
```

Comment:
I don't have the full context, but since the result is different than before, and you still need to do some calculation before using it, what do you think of calling this function `getRawAvailableOptions` to easily indicate that they are raw still?